config/locales/team.yml

```
@@ -71,6 +71,16 @@ en:
               url: "https://example.com"
             - name: "linkedin"
               url: "https://linkedin.com"
+              
+        - name: "MacDaniel Maisel"
+          image: "https://avatars.githubusercontent.com/u/44100935"
+          location: "California"
+          joined: "2022"
+          socials: 
+            - name: "linkedln"
```

Nit: Just a small typo here:

```suggestion
            - name: "linkedin"
```